pkg/checks/checks.go

```diff
@@ -188,6 +299,44 @@ out:
 		}
 	}
 
+	// generate a list of the expected static pods
+	pods := make(map[string]struct{ Namespace string })
```

Comment:
Let's handle static pods as a follow-up.  I'm not entirely sure if we want it or not.